docs/en-US/component/cascader.md

```diff
@@ -144,28 +144,29 @@ cascader/panel
+| separator                         | option label separator                                                                                                                                                           | string                  | —                           | ' / '     |
+| filterable                        | whether the options can be searched                                                                                                                                              | boolean                 | —                           | —         |
+| filter-method                     | customize search logic, the first parameter is `node`, the second is `keyword`, and need return a boolean value indicating whether it hits.                                      | function(node, keyword) | —                           | —         |
+| debounce                          | debounce delay when typing filter keyword, in milliseconds                                                                                                                       | number                  | —                           | 300       |
+| before-filter                     | hook function before filtering with the value to be filtered as its parameter. If `false` is returned or a `Promise` is returned and then is rejected, filtering will be aborted | function(value)         | —                           | —         |
+| popper-class                      | custom class name for Cascader's dropdown                                                                                                                                        | string                  | —                           | —         |
+| teleported                        | whether cascader popup is teleported                                                                                                                                             | boolean                 | —                           | true      |
+| popper-append-to-body(deprecated) | whether to append the popper menu to body. If the positioning of the popper is wrong, you can try to set this prop to false                                                      | boolean                 | —                           | true      |
+| tag-type                          | tag type                                                                                                                                                                         | string                  | success/info/warning/danger | info      |
+| suffix-icon                       | Custom suffix icon component                                                                                                                                                     | `string \| Component`   | —                           | ArrowDown |
```

Comment:
**Suggestion:** I think we should put this line after `validate-event` to keep the change as small as possible, WDYT?

Reply:
OK